backend/src/Edna/Util.hs

```diff
@@ -170,21 +168,15 @@ schemaOptions :: SchemaOptions
 schemaOptions = fromAesonOptions ednaAesonWebOptions
 
 -- | Default implementation of 'ToSchema' via Generics.
-gDeclareNamedSchema
-    :: ( Generic a
-       , S.GToSchema (G.Rep a)
-       , GenericHasSimpleShape a "genericDeclareNamedSchemaUnrestricted" (GenericShape (G.Rep a))
-       )
-    => Proxy a -> Declare (S.Definitions S.Schema) S.NamedSchema
-gDeclareNamedSchema = S.genericDeclareNamedSchema schemaOptions
+gDeclareNamedSchema :: (Generic a, O.GToSchema (G.Rep a))
+                    => Proxy a
+                    -> Declare (O.Definitions O.Schema) O.NamedSchema
+gDeclareNamedSchema = O.genericDeclareNamedSchema schemaOptions
```

Comment:
btw, it's a lot easier to use `DerivingVia` for that: https://github.com/biocad/web-template/blob/master/src/Web/Template/Servant/Aeson.hs#L14

I don't suggest doing anything like that now, that would require changing a lot of code, I'm only demonstrating a concept.
cc @gromakovsky